Spec/Auth.swift

```diff
@@ -3619,5 +3619,331 @@ class Auth : QuickSpec {
                 }
             }
         }
+
+        describe("JWT and realtime") {
+            let channelName = "test_JWT"
+            let messageName = "message_JWT"
+            
+            context("client initialized with a JWT token in ClientOptions") {
+                let options = AblyTests.clientOptions()
+
+                context("with valid credentials") {
+                    options.token = getJWTToken()
+                    let client = AblyTests.newRealtime(options)
+                    defer { client.dispose(); client.close() }
+
+                    it("pulls stats successfully") {
+                        waitUntil(timeout: testTimeout) { done in
+                            client.stats{ stats, error in
+                                expect(error).to(beNil())
+                                done()
+                            }
+                        }
+                    }
+                }
+
+                context("with invalid credentials") {
+                    options.token = getJWTToken(invalid: true)
+                    options.autoConnect = false
+                    let client = AblyTests.newRealtime(options)
+                    defer { client.dispose(); client.close() }
+
+                    it("fails to connect with reason 'invalid signature'") {
+                        waitUntil(timeout: testTimeout) { done in
+                            client.connection.once(.failed) { stateChange in
+                                expect(stateChange!.reason!.code).to(equal(40144))
+                                expect(stateChange!.reason!.description).to(contain("invalid signature"))
+                                done()
+                            }
+                            client.connect()
+                        }
+                    }
+                }
+            }
+
+            // RSA8g RSA8c
+            context("when using auth_url") {
```

Comment:
Minor mostly unimportant comment, but this is `authUrl` in iOS, not `auth_url`

Reply:
Done https://github.com/ably/ably-ios/pull/714/commits/7351d465fc5690dfad10464ad6f539223e3ecbb7